src/components/pages/Home.vue

```diff
@@ -133,6 +143,10 @@ export default {
         color: $color-content-text;
         font-family: $font-monospace;
       }
+
+      code:hover {
+        background-color: #ddf;
+      }
```

Comment:
```suggestion
        font-family: $font-monospace;

        &:hover {
          background-color: #ddf;
        }
      }
```
This can be shortened